json-path/src/main/java/com/jayway/jsonpath/internal/JsonContext.java

```diff
@@ -19,8 +19,11 @@
 import static com.jayway.jsonpath.internal.Utils.notNull;
 import static java.util.Arrays.asList;
 
+import java.util.ArrayList;
```

Comment:
Why do you need this? you are not changing this class??

Reply:
Done.